src/kubernetes-ingress-controller/references/feature-gates.md

```diff
@@ -0,0 +1,64 @@
+
+[k8s]:https://kubernetes.io
+[gates]:https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/
+[stages]:https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/#feature-stages
+
+## Feature gates
+
+Below you will find the overviews of features at various maturity levels:
+
+- [Feature gates for graduated or deprecated features](#feature-gates-for-graduated-or-deprecated-features)
+- [Feature gates for Alpha or Beta features](#feature-gates-for-alpha-or-beta-features)
+
+Please read the [Important Notes](#important-notes) section before using any `Alpha` or `Beta` features.
+
+### Important notes
+
+- Most features will be planned and detailed using [Kubernetes Enhancement Proposals (KEP)][k8s-kep]: If you're interested in the development side of features familiarize yourself with our [KEPs][kic-keps]
+- The `Since` and `Until` rows in below tables refer to [KIC Releases][releases]
+- For `GA` features the documentation exists in the main [Kong Documentation][kong-docs], see the [API reference][api-ref] and [Guides][kic-guides]
+
+An additional **warning** for end-users who are reading this documentation and trying to enable `Alpha` or `Beta` features: it is **very important** to understand that features that are currently in an `Alpha` or `Beta` state may **become `Deprecated` at any time** and **may be removed as part of the next consecutive minor release**. This is especially true for `Alpha` maturity features. In other words, **until a feature becomes GA there are no guarantees that it's going to continue being available**. To avoid disruption to your services engage with the community and read the [changelog](https://github.com/Kong/kubernetes-ingress-controller/blob/main/CHANGELOG.md) carefully to track progress. Alternatively **do not use features until they have reached a GA status**.
+
+[k8s-keps]:https://github.com/kubernetes/enhancements
+[kic-keps]:https://github.com/Kong/kubernetes-ingress-controller/tree/main/keps
+[releases]:https://github.com/Kong/kubernetes-ingress-controller/releases
+[kong-docs]:https://github.com/Kong/docs.konghq.com
+[api-ref]: /kubernetes-ingress-controller/latest/references/custom-resources/
+[kic-guides]: /kubernetes-ingress-controller/latest/guides/overview/
+
+### Documentation
+
+If you're looking for documentation for `Alpha` maturity features you can find feature preview documentation the [kubernetes-ingress-controller repository](https://github.com/Kong/kubernetes-ingress-controller/blob/main/FEATURE_PREVIEW_DOCUMENTATION.md)..
+
+Once a feature graduates from `Alpha` to `Beta` maturity these preview docs will be moved to the main [Kong documentation][kong-docs].
+
+[kong-docs]:https://github.com/kong/docs.konghq.com
+
+### Feature gates for graduated or deprecated features
+
+| Feature                    | Default | Stage      | Since | Until |
+|----------------------------|---------|------------|-------|-------|
```

Comment:
Why is this table empty? If we must have a table, then need some placeholder rows so it doesn't break the markdown:

```suggestion
| Feature                    | Default | Stage      | Since | Until |
|----------------------------|---------|------------|-------|-------|
| None                           |         |            |       |       |
```

Reply:
Empty table was a typo 😨 , fixed.